src/component/Person/Person.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ name, age, sex, isMarried, partnerName }) => {
+
+  const personSex = (sex === 'm') ? 'wife' : 'husband';
+
+   return (
+     <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {name}
+    </h2>
+    <p className="Person__age">
+      {age && `I am ${age}`}
+    </p>
```

Comment:
Use conditional rendering for the whole `<p>` tag. No need to create empty html-elements.